xarray_beam/_src/pangeo_forge.py

```diff
@@ -83,10 +84,14 @@ def __init__(
 
     Args:
       pattern: a `FilePattern` describing a dataset.
+      sub_chunks: split each open dataset into smaller chunks. If not set, the
+        transform will return one file per chunk.
```

Comment:
For consistency with `DatasetToChunks`, consider calling this `chunks`.

You might also add a `split_vars` parameter?

Reply:
I will follow up with a future PR to address the `split_vars` suggestion.